community/OM-Operator-Management/cnv-samples/cnv-operator-install/policy-cnv.yaml

```diff
@@ -0,0 +1,179 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: openshift-virtualization
+  namespace: default
+spec:
+  remediationAction: enforce
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: kubevirt-namespace
+        spec:
+          remediationAction: enforce
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: "v1"
+                kind: "Namespace"
+                metadata:
+                  name: "openshift-cnv"
+            
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1beta1
+        kind: OperatorPolicy
+        metadata:
+          name: kubevirt-hyperconverged-operator
+        spec:
+          remediationAction: enforce
+          complianceType: musthave
+          operatorGroup: # optional
+            name: openshift-cnv
+            namespace: openshift-cnv
+            targetNamespaces:
+                - openshift-cnv
+          subscription:
+            channel: stable
+            name: kubevirt-hyperconverged
+            namespace: openshift-cnv
+            #source: redhat-operators
+            #sourceNamespace: openshift-marketplace
+            startingCSV: kubevirt-hyperconverged-operator.v4.17.3
+            #installPlanApproval: Automatic
```

Comment:
Can we remove the commented, optional parameters? Especially `installPlanApproval`, because it's worse than optional: if you include it, the OperatorPolicy will just be noncompliant with an error.

Reply:
Yes, I removed these in my current iteration, after our slack discussion to get the automatic upgrades working.